samples/fromProject.js

```diff
@@ -16,11 +16,9 @@ if (!projectId) {
 
 // [START gceimages_latest_os_from_project]
 const {GCEImages} = require('gce-images');
-const gceImages = new GCEImages();
-gceImages.getLatest(`${projectId}/my-ubuntu-image`, (err, image) => {
-  if (err) {
-    throw err;
-  }
-  console.log(image);
-});
+async function main() {
+  const gceImages = new GCEImages();
+  console.log(await gceImages.getLatest(`${projectId}/my-ubuntu-image`));
```

Comment:
lets do an assignment, then console.log the results

Reply:
Done